src/every_query/eval.py

```
@@ -0,0 +1,100 @@
+        train_cfg.datamodule.config.task_labels_dir = task_labels_dir
+        D = instantiate(train_cfg.datamodule)
+
+        out = trainer.test(model=M, datamodule=D, ckpt_path=cfg.ckpt_path)
+        m = out[0] if out else {}
+
+        rows.append(
+            {
+                "code": code,
+                "code_slug": slug,
+                "bucket": str(cfg.bucket) if cfg.get("bucket") is not None else None,
+                "occurs_auc": float(m.get(f"{split}/occurs_auc")) if m.get(f"{split}/occurs_auc") is not None else None,
```

Ruff (0.14.8)

Undefined name `split`

(F821)